scroll-search.lic

```diff
@@ -63,8 +63,8 @@ class ScrollSearch
 
     # Variables
     @scroll_count = 0
-	@global_scroll_index = if UserVars.scroll_counter.nil? then 0 else UserVars.scroll_counter end
-    @owned_scrolls = if UserVars.owned_scrolls.empty? then {} else UserVars.owned_scrolls end
+	@global_scroll_index = UserVars.scroll_counter.nil? ? 0 : UserVars.scroll_counter
```

Comment:
You could also do this assignment with `@global_scroll_index = UserVars.scroll_counter || 0` (also applies to the next line).